tests/tools/timeout.sh

```diff
@@ -19,6 +19,11 @@ timeout_killer_thread() {
 		local value=$(($(date +%s -d "$value_string") - $(date +%s)))
 		local now_ts=$(date +%s)
 
+		# Triples test time in Windows environment
+		if is_windows_system; then
+			multiplier=$(python3 -c "print(int(5 * $multiplier))")
```

Comment:
Please use bash here.

Reply:
Have to say that other calculation in the same script also used python, and I based on that one.

I wil update the comment, and the two calculations.